### DISCUSSION TITLE
Better Frequency Display placement

### DISCUSSION THREAD

**Discussion by PCBenetton** [gdyuldin/x6100_gui#37] [Ideas]
When changing frequencies the freq display gets bigger for easy off reading, thats awesome. But it makes a bit difficult for zeroing on a singnal using the waterfall, specially for weaker signals, as it blocks part of the spectrum and waterfall... Maybe dislocating the enlarged frequency down or up (right under the S-Meter) would allow for clearer selection.

Thanks!
73 DE PP2PB

**Comment by gdyuldin** [comment]
This is planned. I want to remove space between spectrum and waterfall, move frequency indicator and rework top part of the screen. Need to draw all this things.

**Reply by samhiatt** [reply]
I was glad when I found I could deselect "Mag Freq" option in Settings to disable that frequency display magnification behavior, because I had the same complaints about it obscuring the spectrum display at exactly the point I wanted to see.
